README.md

```diff
@@ -171,6 +171,27 @@ This performs the same initial heating, before running a further 1000 steps (1 p
 When MD is run with heating the trajectory ```NaCl-nvt-T20.0-T300.0-T300.0-traj.xyz``` and statistics ```NaCl-nvt-T20.0-T300.0-T300.0-stats.dat``` files will indicate the heating range and MD temperature (which may be different). With heating and MD trajectories/statistics within the same files.
 
 
+### Phonons
+
+Calculate phonons with a 2x3x4 supercell, after geometry optimization (using the [MACE-MP](https://github.com/ACEsuit/mace-mp):
+
+```shell
+janus phonons --struct tests/data/NaCl.cif --supercell 2 --supercell 3 --supercell 4 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}"
```

Comment:
It's a bit confusing to specify the x-y-z supercell as 3 separate calls to supercell. If there isn't a better way to specify a list, perhps a string like:
```suggestion
janus phonons --struct tests/data/NaCl.cif --supercell 2x3x4 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}"
```
And then split it in code on `x`